cilium/ebpf.io-summit
# feat: add Capture the Flag page

**Describe what changes this pull request brings**
- This pull request brings the Capture the flag page


**Describe any additional changes**
- [x] move components Hero & Register to shared components


**Steps to test**
1. Open [the page](https://deploy-preview-8--ebpf-summit-2021.netlify.app/capture-the-flag)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/128507726-80d81880-ed2c-43aa-a3e8-bfaf251003cd.png)
![image](https://user-images.githubusercontent.com/48465000/128507755-338a3036-4560-4db7-a0f1-ea26cc3b2c8c.png)
![image](https://user-images.githubusercontent.com/48465000/128507807-e0ee0308-94f2-4f77-a873-04ca91be549b.png)



**Checklist**
